custom/enikshay/templates/enikshay/historical_adherence.html

```diff
@@ -0,0 +1,141 @@
+                    <tr><td><strong>Treatment Phase (IP or CP):</strong></td><td>{{ treatment_phase }}</td></tr>
+                    <tr><td><strong>Adherence Schedule</strong></td><td>{{ adherence_schedule }}</td></tr>
+                    <tr><td><strong>Doses:</strong></td><td>{{ doses }}</td></tr>
+                    <tr><td><strong>Patient Type</strong></td><td>{{ patient_type }}</td></tr>
+                </tbody>
+            </table>
+        </div>
+        <div class="col-md-1"></div>
+    </div>
+
+
+    <style>
+        .cal td {
+            height: 110px;
+            position: relative;
+            border-width: 2px !important;
+        }
+        .cal td.hidden-day {
+            border-width: 0px !important;
+        }
+        .cal .bottom-left {
+            position: absolute;
+            bottom: 0px;
+            left: 0px;
+            max-width: 50%;
+            padding: 5%;
+        }
+        .cal .bottom-right {
+            position: absolute;
+            bottom: 0px;
+            right: 0px;
+            max-width: 50%;
+            padding: 5%;
+        }
+        .cal .middle-icon {
+            margin-bottom: 6px;
+        }
+        .cal {
+            border: none;
+        }
+        .cal .icon {
+            width: 32px;
+            max-width: 100%;
+        }
+        .cal div.icon div {
+            {# This ensures that the containing div keeps a 1:1 aspect ratio #}
+            width: 100%;
+            padding-bottom: 100%;
+        }
+    </style>
+
+    <div class="row">
+        <div class="col-md-1"></div>
+        <div class="col-md-10">
+            <h3>Calendar</h3>
+
+            <table class="table table-bordered cal" >
+                <col style="width:14%">
+                <col style="width:14%">
+                <col style="width:14%">
+                <col style="width:14%">
+                <col style="width:14%">
+                <col style="width:14%">
+                <col style="width:14%">
+                <tbody>
+                    {% for week in weeks %}
+                        <tr>
+                        {% for day in week.days %}
+                            {% if day %}
+                                <td>
+                                    <div>
+                                        {% if day.month_string %}
+                                            <span>{{ day.month_string }}</span>
+                                        {% endif %}
+                                        <span class="pull-right">{{ day.day_string }}</span>
+                                    </div>
+                                    {% if day.adherence_image_key %}
+                                        <div class="bottom-left">
+                                            <img class="icon" src="
+                                                {% if day.adherence_image_key == "directly_observed_dose" %}
```

Comment:
Good call, that was me being lazy. I got the images from app's multimedia download, and those were just the names that hq generated/saved them as. I'll fix.